test/utils.js

```diff
@@ -1,18 +1,17 @@
 var React = require('react');
-var {mount} = require('enzyme');
+var { mount } = require('enzyme');
```

Comment:
This is all formatting changes except removing withMockedConsole because it's not needed with the error handler test rewritten